src/Catty/ViewController/Continue&New/MaintainObject/MaintainScript/BrickManager/BrickManager.m

```diff
@@ -370,6 +430,7 @@ - (NSArray*)scriptCollectionCopyBrickWithIndexPath:(NSIndexPath*)indexPath andBr
         IfLogicBeginBrick *ifLogicBeginBrick = nil;
         IfLogicElseBrick *ifLogicElseBrick = nil;
         IfLogicEndBrick *ifLogicEndBrick = nil;
+        NSMutableArray *nestedBricks = [NSMutableArray new];
```

Comment:
Please split up this very large method into several private methods with a meaningful name.